pandas_vet/__init__.py

```diff
@@ -83,6 +83,13 @@ def run(self):
 
 
 def check_import_name(node: ast.Import) -> List:
+    """Check AST for imports of pandas not using the preferred alias 'pd'.
```

Comment:
A couple of small things that should be applied to all your docstrings:
1. indent parameters to the same level as the regular text
2. use just a single space after the parameter name
3. for your return definitions, you don't need to repeat "returns"

See this example for examples of those 

```python
    """Check AST for imports of pandas not using the preferred alias 'pd'.

    Error/warning message to recommend use of 'pd' alias.

    :param node: an AST node of type Import
    :return errors: list of errors of type PD001 with line number and column offset
    """
```